### PR TITLE
Fixing the clearbit error message issue.

### PR DESCRIPTION
**Pull request by seanchoi-dev**
Clearbit errorMessage fix.
Other bug fixed as well.

Before: https://main--bacom--adobecom.hlx.page/drafts/iscandar/fake-gated-offer-1
After: https://main--bacom--adobecom.hlx.page/drafts/iscandar/fake-gated-offer-1?milolibs=faas-fix

Expected result:
image
image

### CODE REVIEW

File: libs/blocks/faas/utils.js [hunk 1]
```diff
@@ -190,40 +199,23 @@ const afterYiiLoadedCallback = () => {
...
+      hideDisplay(elements);
     });
   });
 
   $('.faas-form').on('submit', (event) => {
+    removeExtraColumnDivForNext();
+    const nextFirstInput = document.querySelector('.faas-form.next input:not([type=hidden])');
+    if (nextFirstInput && !nextFirstInput.value) {
+      nextFirstInput.parentElement.classList.add('error');
+      nextFirstInput.parentElement.querySelector('.errorMessage').textContent = 'Business email cannot be blank.';
```
Comment (meganthecoder) [inline libs/blocks/faas/utils.js:211]: What about localization?

**Review by meganthecoder — APPROVED** [review]
Looks good. Just wondering about localization

**Review by rgclayton — APPROVED** [review]

File: libs/blocks/faas/utils.js (hunk 1 above)
Comment (seanchoi-dev) [reply libs/blocks/faas/utils.js]: Good found!, I updated this only for fallback. (There originally put this just for fallback for edge case that can't load error message temporally from FaaS API.)

**Comment by seanchoi-dev** [comment]
Tested localization.
image

**Pull request merged**